packages/Symfony/src/Rector/Class_/MakeCommandLazyRector.php

```diff
@@ -0,0 +1,129 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\Class_;
+
+use PhpParser\Node;
+use PhpParser\Node\Stmt\Class_;
+use Rector\PhpParser\NodeTraverser\CallableNodeTraverser;
+use Rector\Rector\AbstractRector;
+use Rector\RectorDefinition\CodeSample;
+use Rector\RectorDefinition\RectorDefinition;
+
+/**
+ * @see https://symfony.com/doc/current/console/commands_as_services.html
+ */
+final class MakeCommandLazyRector extends AbstractRector
+{
+    /**
+     * @var string
+     */
+    private const COMMAND_CLASS = 'Symfony\Component\Console\Command\Command';
```

Comment:
You should also check the `ContainerAwareCommand` (it's deprecated but still used in legacy systems).

Reply:
Doesn't that inherit from `Symfony\Component\Console\Command\Command`?

Reply:
If you check all parent classes then this okay  ;)

Reply:
It's type-based control, so yes :)